e2e/services/browser.ts

```diff
@@ -22,12 +22,12 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import { chromium, Browser } from "playwright";
-import { env } from "../services";
+import { chromium, Browser } from 'playwright';
+import { env } from '../services';
 
 type ConnectRetryParams = { url?: string; timeout?: number; interval?: number };
 
-const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
Personally, I kinda like omitting the parens if the type is implicit, but I'm fine with staying unified with Kibana's `.prettierrc`.

Reply:
Hmm actually I prefer the same 🤔